orc8r/cloud/go/services/state/protos/indexer.proto

```diff
@@ -34,23 +31,11 @@ service Indexer {
   rpc CompleteReindex(CompleteReindexRequest) returns (CompleteReindexResponse) {}
 }
 
-message GetIndexerInfoRequest {}
-
-message GetIndexerInfoResponse {
-  // version of the indexer's current implementation
-  uint32 version = 1;
-  // state_types is the type of states the indexer will index
-  // utilized for pre-filtering
-  repeated string state_types = 2;
-}
-
 message IndexRequest {
   // states to reindex
   repeated magma.orc8r.State states = 1;
   // network_id of the states
   string network_id = 2;
-  // reporter_hwid is the hwid of the gateway that reported the states
-  string reporter_hwid = 3;
```

Comment:
Not sure if in this service apply, but we normally don't remove fields on the protos, but deprecate it. 

This is to avoid that other gateways may be using a different version of the proto

```
message Foo {
 string old_field = 1 [deprecated=true];
}
```

Reply:
This is good general advice. In this case though since this is an Orc8r-internal service, and we don't store these protos, it's safe to remove the fields 🙂 @wangyyt1013

Reply:
sounds good, just removed the field!